src/components/ui/InfoTooltip/InfoTooltip.tsx

```diff
@@ -26,7 +27,7 @@ const InfoTooltip = ({ content, children }: InfoTooltipProps) => {
         <AnimatePresence>
           {open && (
             <Tooltip.Portal forceMount>
-              <Tooltip.Content sideOffset={5} side="top" asChild>
+              <Tooltip.Content sideOffset={5} side={side ?? 'top'} asChild>
```

Comment:
It might be easier to read if we declare "top" as default param when deconstructing the props.

So, the line 13 would be like:
```tsx
const InfoTooltip = ({ content, children, side = "top" }: InfoTooltipProps) => {
```

And then, this line would be like:

```suggestion
              <Tooltip.Content sideOffset={5} side={side} asChild>
```